Rhinoceros_Engine/Convert/ToRhino.cs

```diff
@@ -433,11 +433,29 @@ public static RHG.Brep ToRhino(this List<BHG.ISurface> surfaces)
 
         /***************************************************/
 
-        [NotImplemented]
         public static RHG.Extrusion ToRhino(this BHG.Extrusion extrusion)
         {
-            // TODO Rhino_Adapter conversion to Extrusion
-            return null;
+            if (!extrusion.Curve.IIsPlanar()) {
+                BH.Engine.Reflection.Compute.RecordError("The provided BHoM Extrusion has a base curve that is not planar.");
+                return null;
+            }
+
+            var planarCurve = extrusion.Curve.IToRhino();
+
+            RHG.Plane curvePlane;
+            planarCurve.TryGetPlane(out curvePlane);
+
+
+            double extrHeight = extrusion.Direction.Length();
+
+            double angle = RHG.Vector3d.VectorAngle(curvePlane.Normal, extrusion.Direction.ToRhino()) * (180 / Math.PI);
+
+            if (angle > 90)
```

Comment:
Maybe should add a check if the direction and the curve are non-parallel within some tolerance? If not, maybe return null? Potentially to use the IsParallel from geometry engine maybe?

Reply:
Done in https://github.com/BHoM/Rhinoceros_Toolkit/pull/204/commits/d24289bd6b966d4ac4e988c3fb8c079c8a04e2a5